iml-agent/src/action_plugins/snapshots/iml_agent__action_plugins__ldev__tests__create.snap

```diff
@@ -0,0 +1,27 @@
+---
+source: iml-agent/src/action_plugins/ldev.rs
+expression: content
+---
+mds1 mds2 MGS zfs:mdt0/mdt0
+mds1 mds2 zfsmo-MDT0000 zfs:mdt0/mdt0
+mds2 mds1 zfsmo-MDT0001 zfs:mdt1/mdt1
+oss1 oss2 zfsmo-OST0000 zfs:ost0/ost0
+oss1 oss2 zfsmo-OST0001 zfs:ost1/ost1
+oss1 oss2 zfsmo-OST0002 zfs:ost2/ost2
+oss1 oss2 zfsmo-OST0003 zfs:ost3/ost3
+oss1 oss2 zfsmo-OST0004 zfs:ost4/ost4
+oss1 oss2 zfsmo-OST0005 zfs:ost5/ost5
+oss1 oss2 zfsmo-OST0006 zfs:ost6/ost6
+oss1 oss2 zfsmo-OST0007 zfs:ost7/ost7
+oss1 oss2 zfsmo-OST0008 zfs:ost8/ost8
+oss1 oss2 zfsmo-OST0009 zfs:ost9/ost9
+oss2 oss1 zfsmo-OST000a zfs:ost10/ost10
+oss2 oss1 zfsmo-OST000b zfs:ost11/ost11
+oss2 oss1 zfsmo-OST000c zfs:ost12/ost12
+oss2 oss1 zfsmo-OST000d zfs:ost13/ost13
+oss2 oss1 zfsmo-OST000e zfs:ost14/ost14
+oss2 oss1 zfsmo-OST000f zfs:ost15/ost15
+oss2 oss1 zfsmo-OST00010 zfs:ost16/ost16
```

Comment:
Oops:
```
[root@mds1 ~]# lctl snapshot_create --fsname zfsmo --name manual
Invalid snapshot config file /etc/ldev.conf at the line 20 'oss2 oss1 zfsmo-OST00010 zfs:ost16/ost16'
```

I guess it should be `zfsmo-OST0010` (minus one `0`).

Reply:
Sure, i'll update